mobile/lib/screens/for_you_page.dart

```diff
@@ -19,6 +22,10 @@ class _ForYouPageState extends State<ForYouPage>
     with SingleTickerProviderStateMixin {
   late TabController _tabController;
   late bool _analytics;
+  final AppService _appService = AppService();
+  late GlobalKey _analyticsTabShowcaseKey;
+  late GlobalKey _kyaTabShowcaseKey;
+  static BuildContext? myContext;
```

Comment:
This might not be necessary?

Reply:
@NoahNsimbe I included these because I was getting an error of context not found, and the showcase for that page would fail to show.